README.md

```diff
@@ -31,32 +31,29 @@ installed).
 
 The user discussion and development of Git take place on the Git
 mailing list -- everyone is welcome to post bug reports, feature
-requests, comments and patches to git@vger.kernel.org (read
+requests, comments, and patches to git@vger.kernel.org (read
 [Documentation/SubmittingPatches][] for instructions on patch submission).
 To subscribe to the list, send an email with just "subscribe git" in
 the body to majordomo@vger.kernel.org. The mailing list archives are
 available at <https://lore.kernel.org/git/>,
-<http://marc.info/?l=git> and other archival sites.
+<http://marc.info/?l=git>, and other archival sites.
 
-Issues which are security relevant should be disclosed privately to
+Issues that are security relevant should be disclosed privately to
```

Comment:
Both are correct, therefore I consider it to be pointless to change from one to the other.